### PR TITLE
Add S7 Keys

### PR DESCRIPTION
**Pull request by CHMEX**
Targeting 30th of June with the slowless of the blocks 40k added instead of 107k

### CODE REVIEW

**Comment by Alrighttt** [comment]
Could you detail the logic of your height estimation please?

**Comment by CHMEX** [comment]
It's 33531 blocks from now

33531 block ago was block 11379243 Jun 5, 2023 8:04:15 AM so 13 days ago HW set to around in 12 -13 days from now should match if blocktime stays the same.
We have something like 100 blocks in 60 Minutes as supposed to the 360 blocks in 60 Minutes

File: src/komodo_validation015.h [hunk 1]
```diff
@@ -74,14 +74,14 @@
...
-#define NUM_KMD_SEASONS 6
+#define NUM_KMD_SEASONS 7
 #define NUM_KMD_NOTARIES 64
 // first season had no third party coins, so it ends at block 0. 
 // second season ends at approx block 4,173,578, please check this!!!!! it should be as close as possible to July 15th 0:00 UTC. 
 // third season ending height is unknown so it set to very very far in future. 
 // Fourth season is ending at the timestamp 1623682800, and the approximate CHIPS block height of that time is of 8207456.
 // fifth season, targeting timestamp 1656077853; averaging 8640 blocks/day
-static const int32_t KMD_SEASON_HEIGHTS[NUM_KMD_SEASONS] = {0, 4173578, 6415000, 8207456, 10275665, 999999999};
+static const int32_t KMD_SEASON_HEIGHTS[NUM_KMD_SEASONS] = {0, 4173578, 6415000, 8207456, 10275665, 11446305 999999999};
```
Comment (cipig) [inline src/komodo_validation015.h:84]: missing `,`
gives compile error
```
In file included from validation.cpp:299:
komodo_validation015.h:84:110: error: expected ‘}’ before numeric constant
   84 | static const int32_t KMD_SEASON_HEIGHTS[NUM_KMD_SEASONS] = {0, 4173578, 6415000, 8207456, 10275665, 11446305 999999999};
      |                                                            ~                                                 ^~~~~~~~~
komodo_validation015.h:84:110: error: expected ‘,’ or ‘;’ before numeric constant
komodo_validation015.h:84:119: error: expected declaration before ‘}’ token
   84 | static const int32_t KMD_SEASON_HEIGHTS[NUM_KMD_SEASONS] = {0, 4173578, 6415000, 8207456, 10275665, 11446305 999999999};
      |
```

Comment (CHMEX) [reply src/komodo_validation015.h]: added thanks!

**Review by cipig — APPROVED** [review]

**Pull request merged**